server/routerlicious/packages/services/src/factory.ts

```diff
@@ -0,0 +1,32 @@
+import { IDbFactory } from "@fluidframework/server-services-core";
+import { Provider } from "nconf";
+import registerDebug from "debug";
+
+const debug = registerDebug("fluid:backend");
+
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export class DbFactoryFactory {
+    static async create(config: Provider): Promise<IDbFactory> {
+        const BACKEND = process.env.DBBACKEND || "MONGODB";
```

Comment:
Why are you using the environment variable directly here? Wouldn't it be better to use the same configuration system that is used for the other settings? I think nconf supports overriding via environment variables anyways.